tests/usbredir_test.go

```diff
@@ -65,9 +66,8 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 
 		var vmi *v1.VirtualMachineInstance
 		BeforeEach(func() {
-			vmi, err = createVMI(virtClient, false)
-			Expect(err).ToNot(HaveOccurred())
-			tests.WaitForSuccessfulVMIStart(vmi)
+			vmi = libvmi.New(libvmi.WithResourceMemory("32Mi"))
```

Comment:
This seems wrong.

Reply:
Can you elaborate Please?
we had a discussion about the 32Mi:
https://github.com/kubevirt/kubevirt/pull/8075#discussion_r919005138

Reply:
Everythin is alright. I missed the false `vmi, err = createVMI(virtClient, false)`. I apologize

Reply:
What do you think about making this 32Mi a named constant. A constant that we can also comment.

Another person will come here, and try to improve, and will be asking why 32 Mi and not 1Mi or 128Mi?